kitsune/search/v2/documents.py

```diff
@@ -162,6 +162,14 @@ def get_queryset(cls):
             .annotate(es_question_num_votes=Count("votes"))
         )
 
+    @classmethod
+    def db_to_es_id(cls, db_id):
+        return "q_{}".format(db_id)
```

Comment:
why do we need to prefix this? Wouldn't a plain ID be sufficient here? This applies to the Answer too.